src/components/FormField/FormField.js

```diff
@@ -41,13 +49,18 @@ const getTextColor = (props: Props) => {
   }
 };
 
-const WrapperLabel = styled.label`
+const WrapperLabel = styled.label.attrs({
+  // default to 30px spacing between FormFields
+  spacing: props => props.spacing || 30,
```

Comment:
I'll move this to `FormField` component `static defaultProps` as it's only one location to add the default of 30 instead of every styled-component.